components/brave_ads/core/internal/account/confirmations/README.md

```diff
@@ -1,5 +1,7 @@
 # Confirmations
 
-Create anonymous [confirmations](https://github.com/brave/brave-browser/wiki/Security-and-privacy-model-for-ad-confirmations).
+Confirmations are part of the Brave Ads advertising model, which is designed to respect user privacy while also rewarding Brave Rewards users for their attention.
```

Comment:
I think it would be good here to say something about the purpose of these requests. Like:

> These anonymous requests allow Brave to produce aggregate statistics for
> advertisers to assess how their campaigns are performing (e.g.
> total number of impressions, click-through rate, conversion rate).

Reply:
Added, thanks